test/integ/base.test.ts

```diff
@@ -42,14 +42,6 @@ if (!process.env.UNISWAP_API || !process.env.ARCHIVE_NODE_RPC) {
   throw new Error('Must set UNISWAP_API and ARCHIVE_NODE_RPC env variables for integ tests. See README');
 }
 
-if (!process.env.PORTION_API_URL) {
```

Comment:
@zhongeric can these be made optional?

Reply:
Yea should be able to, we don't need internal api key and I don't think our integ tests rely on portion url but I could be wrong